backend/packages/Upgrade/src/api/controllers/ExperimentClientController.ts

```diff
@@ -764,19 +764,15 @@ export class ExperimentClientController {
 
   public async getUserDoc(experimentUserId, logger) {
     const experimentUserDoc = await this.experimentUserService.getOriginalUserDoc(experimentUserId, logger);
```

Comment:
had comment on the issue that I posted at same time you put up the PR:

https://github.com/CarnegieLearningWeb/UpGrade/issues/301

Reply:
what happens if `this.experimentUserService.getOriginalUserDoc` errors? It looks like it needs a try catch block.

Reply:
Adding the try catch block.

Reply:
`const experimentUserDoc = await this.getUserDoc(logData.userId, request.logger);`

I have a question on this line also, what is the performance implication of including this internal network call? It will be an extra network call for each endpoint in this controller?

Reply:
We will track this performance optimisation in another ticket.